cli/githubutil.py

```diff
@@ -146,18 +150,110 @@ def list_draft_releases(
     github_cfg_name: str,
     github_repository_owner: str,
     github_repository_name: str,
+    greatest_release_version: CliHint(
+        help=(
+            'Greatest version (according to Semver) up to which (exclusive) '
+            'to consider draft releases, if outdated draft releases are to be determined'
+        ),
+    )=None,
+    list_outdated_draft_releases: bool = False,
 ):
+    '''List all draft releases in a GitHub repository. Optionally, attempt to print only outdated
```

Comment:
nit: optionally filter heuristically for outdated draft releases. The printing itself should always succeed, so stating that we attempt to print is somewhat misleading imo.